x/ownership/internal/types/key.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+const (
+	// ModuleName is the name of the module
+	ModuleName = "ownership"
+
+	// StoreKey to be used when creating the KVStore
+	StoreKey = ModuleName
+
+	// RouterKey to be used for routing msgs
+	RouterKey = ModuleName
+
+	// QuerierRoute to be used for routing
+	QuerierRoute = ModuleName
```

Comment:
let's use those const instead of ModuleName when it make sense.

Reply:
Ok. Given resolve as you already proposed changes in the client.